### DISCUSSION TITLE
fork之后，自己build docker出现错误

### DISCUSSION THREAD

**Discussion by justincnn** [linweiyuan/go-chatgpt-api#223] [General]
docker hub使用token登录，第一次可以，但是第二次就会出错，

请问下，这个是yaml文件的配置问题，或者说获取token有哪些注意事项，

谢谢

![](https://s2.loli.net/2023/07/15/faOYzXjNUD2wsRQ.png)

**Comment by linweiyuan** [comment]
可能要改下不要让每个 main 改动都打包，这个你自己改。。。我也不熟

第三行那个就是自动拉更新的机器人，正常来说应该是你的 docker 账号

或这简单粗暴一点，你随便改点东西提交，那么就是你的账号，就正常打包

**Comment by justincnn** [comment]
> 可能要改下不要让每个 main 改动都打包，这个你自己改。。。我也不熟
> 
> 第三行那个就是自动拉更新的机器人，正常来说应该是你的 docker 账号
> 
> 或这简单粗暴一点，你随便改点东西提交，那么就是你的账号，就正常打包

改用账号密码登录，而非token就没问题了，不知道是github的问题，还是dcker hub的问题：

~~~yaml
      - name: Login to DockerHub
        uses: docker/login-action@v2
        with:
          username: ${{ secrets.DOCKER_USERNAME }}
          password: ${{ secrets.DOCKER_PASSWORD }}
~~~

**Reply by linweiyuan** [reply]
可以
